### PR TITLE
Correctly trim long cyrillic note

### PR DESCRIPTION
**Pull request by nickvergessen**
Create a contact with http://generator.lorem-ipsum.info/_russian as note

### CODE REVIEW

**Review by georgehrke — APPROVED** [review]

File: apps/dav/lib/CalDAV/CalDavBackend.php [hunk 1]
```diff
@@ -2465,11 +2465,10 @@ public function updateProperties($calendarId, $objectUri, $calendarData, $calend
 							if ($this->db->supports4ByteText()) {
 								$value = preg_replace('/[\x{10000}-\x{10FFFF}]/u', "\xEF\xBF\xBD", $value);
 							}
-							$value = mb_substr($value, 0, 254);
 
 							$query->setParameter('name', $property->name);
 							$query->setParameter('parameter', substr($key, 0, 254));
```
Comment (georgehrke) [inline apps/dav/lib/CalDAV/CalDavBackend.php]: parameter is also coming straight from the ics data and should use mb_substr.

Comment (nickvergessen) [reply apps/dav/lib/CalDAV/CalDavBackend.php]: Done

**Review by rullzer — APPROVED** [review]

**Pull request merged**